common/djangoapps/student/views.py

```diff
@@ -1541,7 +1541,7 @@ def user_signup_handler(sender, **kwargs):  # pylint: disable=unused-argument
             log.info(u'user {} originated from a white labeled "Microsite"'.format(kwargs['instance'].id))
 
 
-def _do_create_account(form, custom_form=None):
+def _do_create_account(form, custom_form=None, site=None):
```

Comment:
You can do a global search of the edx-platform codebase for `_do_create_account`. The commands are `create_random_users` and `create_user` under the common/djangoapps/student app.